build/dist

```diff
@@ -145,20 +145,19 @@ HIVE_VERSION=$("$MVN" help:evaluate -Dexpression=hive.version $@ 2>/dev/null\
 
 echo "Building Kyuubi package of version $VERSION against Spark version - $SPARK_VERSION"
 
+SUFFIX="-$NAME"
```

Comment:
can we also fix the comments in this file

Reply:
This PR has been split to several small PRs

Reply:
Please move discussion to #889